announcements/2020-Week-48/announcement.md

```diff
@@ -0,0 +1,20 @@
+# :zap: This Week in Provider-Independent Announcements (November 27, 2020):zap:
+
+For a comprehensive and always-updated view of all new features and changes for apps, tenant clusters, UI, and documentation, please check out [Changes and Releases](https://docs.giantswarm.io/changes/). Below are the highlights.
+
+## Managed Apps
+
+1. [Grafana v0.1.0-alpha1](https://github.com/giantswarm/grafana-app) is now available in the Giant Swarm Catalog. It contains the latest upstream Grafana v7.3.1.
+2. [Cert Manager v2.3.3](https://docs.giantswarm.io/changes/managed-apps/cert-manager-app/v2.3.3/) schedules hook Jobs on master nodes. You can now create AWS clusters with no Node Pools and Cert Manager will start up correctly.
+
+## Web UI
+
+In [Happa](https://docs.giantswarm.io/changes/web-ui/), deleting an organization is now simpler and upgrading to beta versions is now possible.
+
+## Documentation
+
+- The article on the [operational layers](https://docs.giantswarm.io/basics/giant-swarm-operational-layers/) got an overhaul.
```

Comment:
@marians in this section you prefer to link to the doc itself rather than https://docs.giantswarm.io/changes/documentation/docs/v1.18.8/?

Reply:
I think it is better. It's confusing when user clicks and see changelog. too many clicks to read the doc.